src/classlibnative/bcltype/stringnative.cpp

```diff
@@ -53,29 +53,6 @@ FCIMPL1(FC_BOOL_RET, COMString::IsFastSort, StringObject* thisRef) {
 }
 FCIMPLEND
 
-/*===============================IsAscii===============================
-**Action: Call the helper to walk the string and see if we have any high chars.
-**Returns: void.  The appropriate bits are set on the String.
-**Arguments: vThisRef - The string to be checked.
-**Exceptions: None.
-==============================================================================*/
-FCIMPL1(FC_BOOL_RET, COMString::IsAscii, StringObject* thisRef) {
-    FCALL_CONTRACT;
-
-    VALIDATEOBJECT(thisRef);
-    _ASSERTE(thisRef!=NULL);
-    DWORD state = thisRef->GetHighCharState();
-    if (IS_STRING_STATE_UNDETERMINED(state)) {
-        state = (STRINGREF(thisRef))->InternalCheckHighChars();
-        FC_GC_POLL_RET();
-    }
-    else {
-        FC_GC_POLL_NOT_NEEDED();
-    }
-    FC_RETURN_BOOL(IS_ASCII(state)); //This can indicate either high chars or special sorting chars.
```

Comment:
I guess IS_ASCII can be removed as well https://github.com/dotnet/coreclr/blob/b271aff1fa54c1385143f3b45c1bf3af01c901cd/src/vm/object.h#L1044-L1046